### PR TITLE
sshs 1.3.1 (new formula)

### PR DESCRIPTION
**Pull request by quantumsheep**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/sshs.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Sshs < Formula
+  desc "Graphical command-line client for SSH"
+  homepage "https://github.com/quantumsheep/sshs"
+  url "https://github.com/quantumsheep/sshs/archive/refs/tags/1.3.1.tar.gz"
+  sha256 "cc6cdfe6be6f3efea29c8acc5a60ea037a88e445bced1619e7002a851d9bbe88"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
```
Comment (carlocab) [inline Formula/sshs.rb:9]:
```suggestion
```
You don't actually use `gmake`, so this isn't needed.

File: Formula/sshs.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  def install
+    system "make", "build", "VERSION=#{version}", "OUTPUT=#{bin}/sshs"
+  end
+
+  test do
+    assert_equal "sshs version #{version}", shell_output("#{bin}/sshs --version").strip
```
Comment (carlocab) [inline Formula/sshs.rb:16]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

File: Formula/sshs.rb (hunk 1 above)
Comment (quantumsheep) [reply Formula/sshs.rb:9]: I'm using `make`, doesn't this need the `make` formula ?

File: Formula/sshs.rb (hunk 2 above)
Comment (quantumsheep) [reply Formula/sshs.rb:16]: Sadly I don't really know how to test this program differently as it starts a full UI.

File: Formula/sshs.rb (hunk 1 above)
Comment (carlocab) [reply Formula/sshs.rb:9]: No, `make` refers to the one provided by the system. The formula's executable is called `gmake`.
```
❯ brew info make
make: stable 4.3 (bottled)
Utility for directing compilation
[snip]
==> Caveats
GNU "make" has been installed as "gmake".
If you need to use it as "make", you can add a "gnubin" directory
to your PATH from your bashrc like:

    PATH="/usr/local/opt/make/libexec/gnubin:$PATH"
[snip]
```

File: Formula/sshs.rb (hunk 2 above)
Comment (carlocab) [reply Formula/sshs.rb:16]: You can use Ruby's [`PTY` module](https://ruby-doc.org/stdlib-2.6.3/libdoc/pty/rdoc/PTY.html) to interact with a TUI. You can look for examples in this repository by `grep`ping for `require "pty"`.

Comment (quantumsheep) [reply Formula/sshs.rb]: It should be good now

**Comment by carlocab** [comment]
Thanks @quantumsheep!

P.S. Please let @BrewTestBot squash your commits, @dawidd6

**Comment by dawidd6** [comment]
Thought it would be autosquashed when I dispatch the publishing job, weird.

**Comment by carlocab** [comment]
Ah, you need to add `--autosquash` when doing it in the GitHub UI... It gets added by `brew` if you do it from the CLI

**Comment by carlocab** [comment]
#94704

**Comment by quantumsheep** [comment]
@carlocab as the issue seems fixed, should I recreate another PR or can you accept this one?

**Comment by carlocab** [comment]
The issue is unrelated to your PR. You can already do `brew install sshs` (you may need to `brew update` first and make sure your core repository is on the `master` branch): https://github.com/Homebrew/homebrew-core/blob/c192252570b7ee1149da9f42cfc41f094fe3e79e/Formula/sshs.rb

**Comment by quantumsheep** [comment]
Oh nice! Thanks.

**Pull request closed without merging**